src/fu-security-attr.c

```diff
@@ -203,6 +203,598 @@ fu_security_attr_get_name(FwupdSecurityAttr *attr)
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("Low-level hardware debugging has been enabled for this "
+					  "system and the firmware and OS are not secure."));
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_PLATFORM_DEBUG_LOCKED) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(
+			    strs,
+			    /* TRANSLATORS: longer success message (unlikely to see) */
+			    _("Low-level hardware debugging has been locked to prevent misuse."));
+		} else {
+			g_ptr_array_add(
+			    strs,
+			    /* TRANSLATORS: longer failure message */
+			    _("Low-level hardware debugging can been enabled for this system and "
+			      "if enabled, the firmware and OS would not be secure."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_SUPPORTED_CPU) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("This system can be tested for firmware security."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("This system cannot be tested for firmware security as "
+					  "it does not have a supported processor or mainboard."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_AMD_ROLLBACK_PROTECTION) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The system firmware cannot be downgraded to an older "
+					  "legitimate version with known security problems."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("The system firmware can be downgraded to an older "
+					  "legitimate version with known security problems."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+	if (g_strcmp0(appstream_id, FWUPD_SECURITY_ATTR_ID_AMD_SPI_REPLAY_PROTECTION) == 0) {
+		if (fwupd_security_attr_has_flag(attr, FWUPD_SECURITY_ATTR_FLAG_SUCCESS)) {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer success message (unlikely to see) */
+					_("The system firmware cannot be tricked into repeating "
+					  "privileged actions by a malicious user."));
+		} else {
+			g_ptr_array_add(strs,
+					/* TRANSLATORS: longer failure message */
+					_("The system firmware could be tricked into repeating "
+					  "privileged actions by a malicious user."));
+			contact_hw_vendor = TRUE;
+		}
+	}
+
+	/* any postfix */
+	if (contact_hw_vendor) {
+		g_ptr_array_add(strs,
+				/* TRANSLATORS: OEM is the original end manufacturer, i.e. the logo
+				   on the case */
+				_("Ensure you have installed all available firmware updates and if "
+				  "still not fixed contact your OEM vendor."));
```

Comment:
we know who their OEM is.  How about we replace "your OEM vendor" with a proper value from SMBIOS?